dvm.sh

```diff
@@ -636,10 +650,36 @@ dvm() {
         local VERSION_PATH
         VERSION_PATH="$(dvm_version_path ${VERSION})"
 
+
         if [ -d "$VERSION_PATH" ]; then
-          echo "$VERSION is already installed." >&2
-          dvm use "$VERSION"
-          return $?
+          if [ "$VERSION" == "experimental" ]; then
+            local DVM_ARCH
+            local DVM_OS
+            local DOCKER_BINARY
+
+            local checksum_url
+
+            DOCKER_BINARY="docker"
+            DVM_ARCH="$(dvm_get_arch)"
+            DVM_OS="$(dvm_get_os)"
+
+            checksum_url="$DVM_GET_EXPERIMENTAL_DOCKER_MIRROR/$DVM_OS/$DVM_ARCH/docker-latest.sha256"
+
+            local sum
+            sum=`dvm_download -L -s $checksum_url -o - | command awk '{print $1}'`
+
+            if ( dvm_checksum "$VERSION_PATH/docker" $sum ); then
```

Comment:
As far as I know. It's roughly adapted from nvm's but with a different hashing algorithm.